certchain/certmanager/acmeclient/client.go

```diff
@@ -40,6 +41,15 @@ type Client struct {
 	FetchTiming     certmanager.FetchTiming
 }
 
+// Recommended renewal duration for certs. This is duration before next cert
+// expiry.
+// 8 days is recommended duration to start requesting new certs to allow for
+// ACME server outages. It's 6 days + 2 days renewal grace period. 6 days so
```

Comment:
SXG can be valid for 90 days, I think you mean OCSP validity is 7 days. This duration refers to how many days ahead of expiry we should attempt renewing the certs.  6+2 days is a buffer just in case CA or OCSP server goes down, we can have several attempts at it.

Reply:
I meant SXGs. They are only valid up to 7 days ([Link](https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html#section-3.5-7.3)). What can be valid up to 90 days are SXG _certificates_.

Reply:
Sorry, yes we are talking about the same thing.

Reply:
I see what you mean.  I made the base case 7 days and the added 2 days to make the duration 9 days.